assets/src/js/header.js

```diff
@@ -212,6 +212,17 @@ export const setupHeader = () => {
 
   const toggleElements = [...document.querySelectorAll(toggleElementClasses.join(','))];
 
+  /*
+    This logic it's used to remove undesired active class in cases where anchors are part of the pathname.
```

Comment:
According to this [Slack thread](https://greenpeace-gpi.slack.com/archives/G015K63081W/p1668433007183869), we don't need this code
```
  document.querySelectorAll('.nav-submenu .nav-item.active a').forEach((item) => {
    if(item.getAttribute('href') !== window.location.pathname) {
      const className = item.parentNode.getAttribute('class').replace('active', '');
      item.parentNode.setAttribute('class', className)
    }
  });
```